src/bespokelabs/curator/request_processor/batch/anthropic_batch_request_processor.py

```diff
@@ -0,0 +1,224 @@
+import logging
+import litellm
+from litellm import get_max_tokens
```

Comment:
Let's not import this, but use `litellm.get_max_tokens` similar to other places.